lib/appium/AppiumRunner.js

```diff
@@ -107,11 +107,11 @@ class AppiumRunner {
         // we would get the chance to kill appium server
         // jasmine.configureDefaultReporter({ showColors: false });
 
-        const outputDir = this.findTests.options.output || process.cwd();
+        const outputDir = this.options.output || process.cwd();
```

Comment:
Can you merge these changes to the commit where you used this variable `findTests` incorrectly? And for the others below.

Reply:
Yes, I can do this.